go/cmd/vtctldclient/command/permissions.go

```diff
@@ -70,6 +70,15 @@ func commandGetPermissions(cmd *cobra.Command, args []string) error {
 	if err != nil {
 		return err
 	}
+	// Obfuscate the checksum so as not to potentially display sensitive info.
+	if resp != nil && resp.Permissions != nil {
+		for _, up := range resp.Permissions.UserPermissions {
+			if up != nil {
+				up.PasswordChecksum = 0
+			}
+		}
+	}
+	cli.DefaultMarshalOptions.EmitUnpopulated = false
```

Comment:
Now that we're removing it entirely here, we could add whitespace before and after to make it a logical block and change the comment to...
```
// Elide the PasswordChecksum from the response so as not to potentially display sensitive info.
```